### PR TITLE
>6 cargo data vaildation

### PR DESCRIPTION
**Pull request by eckshn**

### CODE REVIEW

File: backend/data_validation/data_val_2022.py [hunk 1]
```diff
@@ -126,3 +134,29 @@ def check_for_missing_shooting_zones(
...
+        :param auto_upper_hub: Number of balls shot into the upper hub during Autonomous.
+        :param auto_misses: Number of balls missed when shooting during Autonomous.
+        :return:
+        """
+        # Checks auto ball shot count
+        balls_shot_in_auto = auto_lower_hub + auto_upper_hub + auto_misses
+        if balls_shot_in_auto > 6:
+            self.add_error(
+                f"In {match_key}, {team_number} INCORRECT AUTO SHOT COUNT",
+                error_type=ErrorType.INCORRECT_DATA,
```
Comment (sswadkar) [inline backend/data_validation/data_val_2022.py]: i don't think this is INCORRECT_DATA it should be a warning (bc anything higher than 6 is unlikely but still possible).

File: backend/data_validation/data_val_2022.py [hunk 2]
```diff
@@ -126,3 +134,29 @@ def check_for_missing_shooting_zones(
...
+        :param auto_lower_hub: Number of balls shot into the lower hub during Autonomous.
+        :param auto_upper_hub: Number of balls shot into the upper hub during Autonomous.
+        :param auto_misses: Number of balls missed when shooting during Autonomous.
+        :return:
+        """
+        # Checks auto ball shot count
+        balls_shot_in_auto = auto_lower_hub + auto_upper_hub + auto_misses
+        if balls_shot_in_auto > 6:
+            self.add_error(
+                f"In {match_key}, {team_number} INCORRECT AUTO SHOT COUNT",
```
Comment (sswadkar) [inline backend/data_validation/data_val_2022.py]: it's not incorrect as much as it is unlikely

File: backend/data_validation/tests/test_data_val_2022.py [hunk 3]
```diff
@@ -4,35 +4,35 @@
...
-    final_climb_type: str = "Traversal",
-    defense_pct: float = 1.0,
-    defense_rating: int = 5,
-    counter_defense_pct: int = 0,
-    counter_defense_rating: int = 0,
-    driver_rating: int = 5,
-    auto_notes: str = "",
-    teleop_notes: str = "",
-    misc_notes: str = "",
+        scout_id: str = "falcon",
```
Comment (sswadkar) [inline backend/data_validation/tests/test_data_val_2022.py]: unindent all of these?

**Review by sswadkar — CHANGES_REQUESTED** [review]
lgtm just a few syntax changes

**Comment by sswadkar** [comment]
oh also pls run the lint stuff
this should be done on precommit but if it says `failed` a bunch just `git add .` `git commit -m <old commit message>` and then it should fix everything for you

**Comment by eckshn** [comment]
> oh also pls run the lint stuff this should be done on precommit but if it says `failed` a bunch just `git add .` `git commit -m <old commit message>` and then it should fix everything for you

lowkey idk what lint is and my python is wacky

**Pull request merged**